package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yn",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Don't bump the version in a PR. Authors have their own way of releasing. I've fixed it for you :)